ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -691,6 +690,12 @@ const NetworksForm = ({
         />
         <FormField
           warning={warnings.ticker?.msg || ''}
+          warningProps={{
+            'data-testid': 'network-form-ticker-warning',
+            style: {
+              color: 'var(--color-warning-default)',
+            },
```

Comment:
Would you be able to use the Text color prop instead? 

```suggestion
            color: TextColor.warningDefault
```

Reply:
<img width="245" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/be437863-b781-4c96-b9c7-40bd327f9f6c">

Normally yes, but the TextColor.warningDefault class is getting overwritten by the deprecated form-field's h6 color of TextColor.textAlternative. I used the inline style to override the FormField default color.

Also I want to change this component to use FormTextField if possible, but I think that should be done in a new PR.